dbaccess/transaction.go

```diff
@@ -73,7 +73,8 @@ func TransactionsByAddress(ctx Context, address string, order Order, skip uint64
 		Where("`out_addresses`.`address` = ?", address).
 		Or("`in_addresses`.`address` = ?", address).
 		Limit(limit).
-		Offset(skip)
+		Offset(skip).
+		Group("transactions.id")
```

Comment:
The more correct way is to use SELECT DISTINCT if you are not doing any aggregates.
In Gorm you'll have to do `.Select("DISCTINCT transactions.id")`